iroh-sync/src/ranger.rs

```diff
@@ -40,13 +40,13 @@ pub trait RangeEntry: Debug + Clone {
 
 /// A trait constraining types that are valid entry keys.
 pub trait RangeKey: Sized + Debug + Ord + PartialEq + Clone + 'static {
-    /// Returns `true` if `self` is a prefix of `other`.
-    fn is_prefix_of(&self, other: &Self) -> bool;
+    // /// Returns `true` if `self` is a prefix of `other`.
+    // fn is_prefix_of(&self, other: &Self) -> bool;
 
-    /// Returns true if `other` is a prefix of `self`.
-    fn is_prefixed_by(&self, other: &Self) -> bool {
-        other.is_prefix_of(self)
-    }
+    // /// Returns true if `other` is a prefix of `self`.
+    // fn is_prefixed_by(&self, other: &Self) -> bool {
+    //     other.is_prefix_of(self)
+    // }
```

Comment:
@Frando What is the deal with this trait?

Reply:
oh, it's used in tests

Reply:
@Frando I think I've made this happy for the compiler.  But could you make sure it does the right thing?